content/articles/data-representation-using-brython/index.md

```diff
@@ -0,0 +1,345 @@
+
+##### Chart.py file
+The chart.py file contains the following code:
+
+```python
+from browser import window
+
+b_highchart = window.Highcharts.Chart.new
+b_highchart(
+    {
+        "chart": {
+            "plotBackgroundColor": "#FCFFC5",
+            "plotBorderWidth": None,
+            "plotShadow": False,
+            "renderTo": "container",
+        },
+        "title": {"text": "Religious Distribution in Nigeria"},
+        "tooltip": {"pointFormat": "<b>{point.percentage:.1f}%</b>"},
+        "plotOptions": {
+            "pie": {
+                "allowPointSelect": True,
+                "cursor": "pointer",
+                "dataLabels": {
+                    "enabled": True,
+                    "format": "<b>{point.name}</b>: {point.percentage:.1f} %",
+                    "style": {"color": "black"},
+                },
+            }
+        },
+        "series": [
+            {
+                "type": "pie",
+                "data": [
+                    ["Indigenious Beliefs", 10.7],
+                    ["Other Christian", 35.3],
+                    {
+                        "name": "Roman Catholicism",
+                        "y": 10.0,
+                        "sliced": True,
+                        "selected": True,
+                    },
+                    ["Islam", 43.5],
+                    ["Others", 0.5],
+                ],
+            }
+        ],
+    }
+)
+```
+The code above plots the pie chart using the following methods:
+- `from browser import window`: imports the window module from, thus allowing Brython access to Javascript object.
+- `plotBackgroundColor`: It plots the background color for the chart area.
+- `plotBorderWidth`: It plots the width of the chart area in pixels.
+- `plotShadow`: This method takes in boolean values and adds a drop shadow to the area. 
+- `renderTo`: This method is like `document.getElementBy` in javascript. It will take the `id` of the `div` tag in our HTML code.
+- `tooltip`: This object contains the details in the small box when we hover on our chart.
+- `allowPointSelect`: If set as `true`, it allows you to toggle or select any of the slices in the pie chart
+- `Series`: this is a set of data. In this case, it is the religious distribution of people in Nigeria.
+
+When we execute this code, our pie chart is as shown below:
+
+![piechart1](engineering-education/data-representation-with-react-and-ant-design/piechart1.png) 
+
+![piechart2](engineering-education/data-representation-with-react-and-ant-design/piechart2.png)
+
+#### Creating a scatterplot using Brython.
+A scatterplot tests the relationship between two factors. To achieve this, we will put in place the HTML file used for the pie chart. The only difference is the code in the chart.py file.
+
```

Comment:
Please reorganize this section. This section is hard to follow. This means we have some steps that are missing or the steps are not well explained in chronological order.

For example, in beginning, you have said "Having created our chart.py file"... but there is no chart.py file we have created. You have created the chart.py file later on in the article. First, create the chart.py' file. Then you can import/add this file into your HTML boilerplate. When creating the chart.py` file make sure you explain the attributes/methods used.

Next, you can now start explaining the HTML file. Explain which files you have added to the HTML file like the chart.py, brython.js, brython_stdlib.js, and other files. I believe this HTML file is the final part that holds all the imported files.

You can not start explaining this HTML file and you have not explained how you have created the imported files.

Finally, you can now say that using the HTML file we will be able to create a pie chart.

Just ensure we have a chronologic order of the steps that a reader can easily follow.